infrastructure/base/main.tf

```diff
@@ -74,10 +49,85 @@ data "aws_subnets" "subnets_with_ec2_instance_type_offering_map" {
   }
 }
 
+module "iam" {
+  source = "./modules/iam"
+}
+
+resource "random_password" "api_token_salt" {
+  length           = 32
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "random_password" "admin_jwt_secret" {
+  length           = 32
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "random_password" "transfer_token_salt" {
+  length           = 32
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+resource "random_password" "jwt_secret" {
+  length           = 32
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
 locals {
-  subnets_with_ec2_instance_type_offering_ids = sort([for k, v in data.aws_subnets.subnets_with_ec2_instance_type_offering_map : v.ids[0]])
+  subnets_with_ec2_instance_type_offering_ids = sort([
+    for k, v in data.aws_subnets.subnets_with_ec2_instance_type_offering_map : v.ids[0]
+  ])
+  staging_cms_env                             = {
```

Comment:
Is it possible to move all this to Staging (env) module when it is Stage related? I guess same stuff will be here also for Production at future so kind of keeping it DRY

Reply:
yup, good catch